web/src/user_profile.js

```diff
@@ -798,6 +798,16 @@ export function show_edit_user_info_modal(user_id, $container) {
         user_deactivation_ui.confirm_reactivation(user_id, handle_confirm, true);
     });
 
+    $("#edit-user-form #edit_user_full_name").on("input", () => {
+        const $full_name = $("input[name='full_name']");
```

Comment:
Using this selector does not work. It would be better to use `$("#edit_user_full_name")`.